sdk/storage/storage-queue/src/QueueClient.ts

```diff
@@ -538,6 +730,254 @@ export class QueueClient extends StorageClient {
     }
   }
 
+  /**
+   * Clear deletes all messages from a queue.
+   * @see https://docs.microsoft.com/en-us/rest/api/storageservices/clear-messages
+   *
+   * @param {MessagesClearOptions} [options] Options to Messages clear operation.
+   * @returns {Promise<Models.MessageClearResponse>} Response data for the Messages clear operation.
+   * @memberof QueueClient
+   */
+  public async clearMessages(
+    options: MessagesClearOptions = {}
+  ): Promise<Models.MessagesClearResponse> {
+    const { span, spanOptions } = createSpan("QueueClient-clearMessages", options.spanOptions);
+    try {
+      return this.messagesContext.clear({
+        abortSignal: options.abortSignal,
+        spanOptions
+      });
+    } catch (e) {
+      span.setStatus({
+        code: CanonicalCode.UNKNOWN,
+        message: e.message
+      });
+      throw e;
+    } finally {
+      span.end();
+    }
+  }
+
+  /**
+   * Enqueue adds a new message to the back of a queue. The visibility timeout specifies how long
+   * the message should be invisible to Dequeue and Peek operations.
+   * The message content is up to 64KB in size, and must be in a format that can be included in an XML request with UTF-8 encoding.
+   * To include markup in the message, the contents of the message must either be XML-escaped or Base64-encode.
+   * @see https://docs.microsoft.com/en-us/rest/api/storageservices/put-message
+   *
+   * @param {string} messageText Text of the message to enqueue
+   * @param {MessagesEnqueueOptionas} [options] Options to Messages enqueue operation.
+   * @returns {Promise<Models.MessagesEnqueueResponse>} Response data for the Messages enqueue operation.
+   * @memberof QueueClient
+   */
+  public async enqueueMessage(
+    messageText: string,
+    options: MessagesEnqueueOptions = {}
```

Comment:
The name `MessagesEnqueueOptions` was for the `enqueue` method on `MessagesClient`.
Now that it is moved to `QueueClient` and the method is `enqueueMessage`, should this be renamed to `EnqueueMessageOptions`?

 `MessagesEnqueueOptions` -> `EnqueueMessageOptions` or `QueueEnqueueMessageOptions`.

Same is applicable for many other options/responses interfaces in this file.

Reply:
We will have to rename these for consistency. Let's leave them as is for now.